src/pages/login/tests/login-page.spec.js

```diff
@@ -59,11 +63,33 @@ describe('Login page test', () => {
   });
 
   it('Should render Loader', () => {
-    storage = {
-      loginError: '',
-      userLoading: true
-    };
+    storage.userLoading = true;
     const component = shallow(<Login />);
     expect(component.exists(Loader)).toBe(true);
+    storage.userLoading = false;
+  });
+
+  it('should simulate change', async () => {
+    const component = shallow(<Login />);
+    const inputEmail = component.find('#email');
+    const value = { target: { value: 'Hello' } };
+    inputEmail.simulate('change', value);
+    expect(mockChange).toHaveBeenCalledWith(value);
+  });
+
+  it('should simulate click', () => {
+    const component = shallow(<Login />);
```

Comment:
If you are getting components in every test, maybe it is a good idea to make it beforeEach?